contrib/completion/bash/docker

```diff
@@ -4729,10 +4729,10 @@ _docker_trust() {
 	esac
 }
 
-_docker_trust_revoke() {
+_docker_trust_inspect() {
 	case "$cur" in
 		-*)
-			COMPREPLY=( $( compgen -W "--help --yes -y" -- "$cur" ) )
```

Comment:
Diff view on github looks a bit odd here, because the order of commands was changed (they're alphabetical, so order changed due to `view` -> `inspect`)